extensions/fix-language/package.json

```diff
@@ -0,0 +1,64 @@
+{
+  "$schema": "https://www.raycast.com/schemas/extension.json",
+  "name": "fix-language",
+  "title": "Fix Language",
+  "description": "Switch the language input source of the text when you type and forget to change the language.",
+  "icon": "command-icon.png",
+  "author": "tin5451",
+  "categories": [
+    "Productivity",
+    "Developer Tools",
+    "Other"
+  ],
+  "license": "MIT",
+  "commands": [
+    {
+      "name": "sw",
+      "title": "Sw",
```

Comment:
Could we improve the naming?

Reply:
Sure, I just need something short that I can type quickly when using. Do you have any suggestions for a name or can it be an alias? The name I have in mind is `Switch Selected Language`. and `Switch Clipboard Language` comments below.